src/mstabs.h

```diff
@@ -22,36 +21,47 @@ void cpmstabs( f77int *, f77int *, char (*)[4], char (*)[12], char (*)[4],
         char (*)[24], char (*)[8], char (*)[120], f77int *, f77int *,
         char (*)[120], char (*)[8], f77int *, f77int *, f77int * );
 
-#define MSTABS_BASE(var) mstabs_newCarr_ ## var
+/** ??? @return ??? */
+#define MSTABS_BASE(var) mstabs_newCarr_ ## var 
 
```

Comment:
`MSTABS_BASE` is a macro for text substitution within this file.  In other words, it's not a function, so it doesn't have a return value, and neither do any of the below instances where this macro is used.

Reply:
Yes I had to insert those to get doxygen to stop complaining. I will circle back around and see how to better get doxygen to document this macro.